vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc

```diff
@@ -49,7 +49,7 @@ export CTLPLANE_KUBEADM_EXTRA_CONFIG="
       - retrieve.configuration.files.sh \"https://raw.githubusercontent.com/kubernetes/release/{{ KUBERNETES_BINARIES_CONFIG_VERSION }}/cmd/kubepkg/templates/latest/deb/kubeadm/10-kubeadm.conf\" /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
       - usermod -aG docker {{ IMAGE_USERNAME }}
       - systemctl enable --now docker keepalived kubelet
-      - systemctl link /etc/systemd/system/multi-user.target.wants/monitor.keepalived.service
+      - systemctl link /lib/systemd/system/monitor.keepalived.service
```

Comment:
Sorry if I missed the point, but why is this change needed?

Reply:
the link creation failed when writing under /etc/. /lib/ is the "natural" place for service items (other service itmes are there too)